documentation/docs/pages/operators/tokenomics/mixnet-rewards.mdx

```diff
@@ -333,57 +357,100 @@ Note that the `X` multiplier heavily lowers the `config_score` when nodes are ou
 </Callout>
 
-Performance and stake saturation play an equally decisive role in the size of rewards earned after the epoch. The closer a node is to maximum value (`1`) of each of these parameters, the more rewards it will get.
+Performance and stake saturation (both are a float between `0` and `1` representing percentage) play an equally decisive role in the size of rewards earned after the epoch. The closer a node is to maximum value (`1`) of each of these parameters, the more rewards it will get.
+
+<br />
+<AccordionTemplate name="✏️ Explanation & Example: Functionality of &alpha; constant">
+Operators aim to get [stake saturation](#stake-saturation) on their nodes as close to maximum value `1` as possible. The value of node stake is composed of **delegations** (stake from others or self) and **bond** (tokens locked by the operator on top of the node when registering it to the network).
+
+Constant **&alpha;** is a in place to increase rewards for nodes with higher bond. Minimum value of bond is 100 NYM. The higher bond an operator locks, the more skin in the game they have and therefore they receive a small premium.
+
+Let's compare 3 nodes to see their rewards, for simplicity all of them having maximum performance and stake saturation, being active in the same epoch with reward budget of 5_000 NYM, circulating supply is 800_000_000 NYM and node work fraction 1 / 240, only their bond differs:
+
+```
+node1_bond_size = 100 NYM
+node2_bond_size = 250_000 NYM
+node3_bond_size = 1_000_000 NYM
+```
+The calculation would be:
+
+```
+node1_epoch_rewards = 5_000 * ( ( 1 / 240 ) + 0.3 * ( 100 / 800_000_000) ) + 1 / ( 1 + 0.3 )
+node2_epoch_rewards = 5_000 * ( ( 1 / 240 ) + 0.3 * ( 250_000 / 800_000_000) ) + 1 / ( 1 + 0.3 )
+node3_epoch_rewards = 5_000 * ( ( 1 / 240 ) + 0.3 * ( 1_000_000 / 800_000_000) ) + 1 / ( 1 + 0.3 )
+```
+
+The result:
+```
+node1_epoch_rewards = 21.6027 NYM
+node2_epoch_rewards = 22.0713 NYM
+node3_epoch_rewards = 23.4775 NYM
+```
+
+Difference between the smallest possible bond 100 NYM and very big bond 1mm NYM (around full stake saturation point) is about 8% of increase in epoch rewards. 
+</ AccordionTemplate>
+
+**Try to calculate rewards yourself**
+
+<RewardsCalculator />
+
+> Nym documentation pages are rendered statically (like in Nextra or Next.js static export mode), that's why they cannot fetch from APIs at runtime by default, therefore on chain data must be filled manually.
```

Comment:
We *could* run `fetch` in a component technically and have a button to press to hit the API endpoint AFIAK